tests/integration/release/parallel_test_launcher.py

```diff
@@ -153,8 +151,10 @@ def parse_outputs(output_dir: str, job_ids: list[str] | None = None) -> None:
             outcome = "not_completed"
         elif "failed" in last_line.lower():
             outcome = "failed"
-        else:
+        elif "passed" in last_line.lower():
             outcome = "passed"
+        else:
+            outcome = "not_completed"
```

Comment:
"not_completed" was already designated in line 151. Should this be some new catch-all like "unresolved"?

Reply:
This happens if tests error out or they all get skipped, which I feel comfortable capture with the case tests_not_completed, but I'm open to other ideas given these specific types of failures.